### PR TITLE
Feat/add navigation structure

### PR DESCRIPTION
**Pull request by vonskareng**
If you think this is useful for the app we can merge it

### CODE REVIEW

**Comment by AdamOstgaard** [comment]
Would be great with some screenshots!

**Comment by AdamOstgaard** [comment]
I really think we should try to write tests before we merge this.

File: pvt71/components/__tests__/AppSingleButton-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import "react-native";
+import React from "react";
+import  AppSingleButton  from "../AppSingleButton";
```
Comment (mausworks) [inline pvt71/components/__tests__/AppSingleButton-test.js]: NIT: Extra space after import

File: pvt71/screens/SettingsScreen.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-    title: 'app.json',
-  };
-
   render() {
-    /* Go ahead and delete ExpoConfigView and replace it with your
-     * content, we just wanted to give you a quick view of your config */
-    return <ExpoConfigView />;
+    return (
+      <View style={styles.container}>
+        <Text>SettingsScreen</Text>
```
Comment (mausworks) [inline pvt71/screens/SettingsScreen.js]: Should this not be human readable? I.e. "Settings screen"? Or just... settings?

**Comment by AdamOstgaard** [comment]
I'm going to approve this pull request even though there are unresolved discussions as we have to pick up the development speed.

**Comment by AdamOstgaard** [comment]
Pull master into this branch locally and regenerate the snapshots to merge it.

**Pull request merged**